sdk/nodejs/project.json

```diff
@@ -25,6 +25,12 @@
             }
         },
         "test": {
+            "executor": "nx:run-commands",
+            "options": {
+                "command": "NODE_OPTIONS=--experimental-vm-modules yarn nx test-jest nodejs",
```

Comment:
@ajwootto note this is what I had to do to get the jest tests to work with the dynamic import.